src/main/java/appeng/block/grindstone/BlockCrank.java

```diff
@@ -164,4 +166,16 @@ public boolean isFullCube( IBlockState state )
 		return false;
 	}
 
+	@Override
+	public boolean canPlaceTorchOnTop( IBlockState state, IBlockAccess world, BlockPos pos )
+	{
+		return false;
+	}
+
+	@Override
+	public BlockFaceShape getBlockFaceShape( IBlockAccess worldIn, IBlockState state, BlockPos pos, EnumFacing face )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)